<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkClass</class>
 <widget class="QWidget" name="LinkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Link</string>
  </property>
  <widget class="QPushButton" name="buttonConnect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>210</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxInstrument">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>460</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDisconnect">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>210</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
